src/components/webgl/WebglBox.tsx

```diff
@@ -18,8 +18,9 @@ function WebglBox ({ json }: { json: any }): JSX.Element {
 
     useEffect(function () {
         unityContext.on("loaded", function () {
+            console.log("2 ->" + json);
```

Comment:
do we still need this console.log? If yes, what is it for?

Reply:
No, it can be removed.

Reply:
So i'll delete this one, and the others. then i'm fine with the pr. Just make sure that the others all also fine with it. Mainly because, we worked on this together, so i have more insight of the code it self.